Formula/mongodb-atlas-cli.rb

```diff
@@ -1,8 +1,8 @@
 class MongodbAtlasCli < Formula
   desc "Atlas CLI enables you to manage your MongoDB Atlas"
   homepage "https://www.mongodb.com/docs/atlas/cli/stable/"
-  url "https://github.com/mongodb/mongodb-atlas-cli/archive/refs/tags/atlascli/v1.10.0.tar.gz"
-  sha256 "d005e51d610526c7dec4623ce4c9f2975f7c0d96a7f1548cd8f5f36188cf08c0"
+  url "https://github.com/mongodb/mongodb-atlas-cli/archive/refs/tags/mongocli/v1.31.0.tar.gz"
```

Comment:
this is wrong, this is for mongocli

Reply:
the livecheck would also report error

```
$ brew livecheck mongodb-atlas-cli
mongodb-atlas-cli: 1.10.0 ==> 1.10.0
```

Reply:
see https://github.com/Homebrew/homebrew-core/pull/131959

Reply:
- actually this, #139262

you can use `brew bump --open-pr mongocli` next time

Reply:
Thank you.